examples/ads.amp.html

```diff
@@ -906,6 +907,18 @@ <h2 id="yieldmo">Yieldmo</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="webediads">Webediads (private ad network)</h2>
```

Comment:
@Lith if so, could you add a line calling out that
```
It's a private ad network with strict ad targeting, hence very common to see a no-fill.
```

So that the other contributor would not be surprised it's not showing ads properly.